src/core/config.go

```diff
@@ -21,9 +21,10 @@ type Project struct {
 
 // Jira define jira params
 type Jira struct {
-	Username string
-	Password string
-	URL      string
+	Username     string
+	Password     string
+	URL          string
+	ClosedStatus string
```

Comment:
What about `ClosedStatuses` with a slice of strings, as it holds multiple statuses?